config-windows.md

```diff
@@ -142,11 +138,15 @@ You can indicate that a container should be started in an a mode where disk flus
 
 ## <a name="configWindowsHyperV" />HyperV
 
-`hyperv` is an OPTIONAL field of the Windows configuration. If present, the container MUST be run with Hyper-V isolation. If omitted, the container MUST be run as a Windows Server container.
+`hyperv` is an OPTIONAL field of the Windows configuration.
+If present, the container MUST be run with Hyper-V isolation.
+If omitted, the container MUST be run as a Windows Server container.
 
 The following parameters can be specified:
 
-* **`utilityvmpath`** *(string, OPTIONAL)* - specifies the path to the image used for the utility VM. This would be specified if using a base image which does not contain a utility VM image. If not supplied, the runtime will search the container filesystem layers from the bottom-most layer upwards, until it locates "UtilityVM", and default to that path.
+* **`utilityvmpath`** *(string, OPTIONAL)* - specifies the path to the image used for the utility VM.
+    This would be specified if using a base image which does not contain a utility VM image.
+    If not supplied, the runtime will search the container filesystem layers from the bottom-most layer upwards, until it locates "UtilityVM", and default to that path.
 
 * **`sandboxpath`** *(string, REQUIRED)* - specifies the root of the path to the sandbox to be used for the container.
```

Comment:
Looks like https://github.com/opencontainers/runtime-spec/pull/849/files is getting ready to remove this, you might have some merge fun between the two PR's.